server/util/live-loan/live-loan-draw.js

```diff
@@ -0,0 +1,130 @@
+const path = require('path');
+const { createCanvas, registerFont, loadImage } = require('canvas');
+const { ellipsisLine, wrapText, roundRect } = require('./canvas-utils');
+const loanUseFilter = require('../../../src/plugins/loan-use-filter.common'); // Duplicate of the es module version
+
+module.exports = loanData => {
+	return new Promise((resolve, reject) => {
+		async function draw() {
+			const resizeFactor = 3;
+			const borderThickness = 2 * resizeFactor;
+			const cardWidth = 300 * resizeFactor;
+			const cardHeight = 525 * resizeFactor;
+			const borrowerImgAspectRatio = 0.75;
+			const kivaColors = {
+				blue: '#118aec',
+				darkBlue: '#006ed3',
+				green: '#4faf4e',
+				textLight: '#999999',
+				strokeGrey: '#d8d8d8',
+				textDark: '#484848',
+				white: '#ffffff'
+			};
+
+			function fontFile(name) {
+				return path.join(__dirname, '../../../src/assets/fonts', name);
+			}
+			registerFont(fontFile('PostGrotesk-Medium.ttf'), { family: 'Kiva Post Grot', weight: '400' });
+			registerFont(fontFile('PostGrotesk-MediumItalic.ttf'), { family: 'Kiva Post Grot', weight: '400', style: 'italic' });
```

Comment:
TODO: Having issues with node-canvas and registering fonts. So far only the first font is able to be used. Still working on this but I don't think it'll be a dealbreaker.

Reply:
What if you used a different font family name for each one?

Reply:
No luck there, I think I may need to patch the fonts or wait for a new release of node-canvas - https://github.com/Automattic/node-canvas/issues/1572